### PR TITLE
prefer the new unittest.mock from the standard library

### PR DESCRIPTION
**Pull request by a-detiste**
https://github.com/testing-cabal/mock

mock is now part of the Python standard library, available as unittest.mock in Python 3.3 onwards.

### CODE REVIEW

File: src/ofxstatement/tests/test_plugin.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import unittest
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
```
Comment (kedder) [inline src/ofxstatement/tests/test_plugin.py:6]: We don't support the ancient versions of Python, so I think this precaution is unnecessary (unless I'm missing something).

**Comment by a-detiste** [comment]
Hi,

The baseline is not explicitly documented.

So yes a plain "from unittest import mock" would be enough in modern system.


Le lun. 10 juin 2024 à 15:02, Andrey Lebedev ***@***.***> a
écrit :

**Comment by kedder** [comment]
It is somewhat documented in pyproject.toml: https://github.com/kedder/ofxstatement/blob/225075416c6a4f4bd9a1dd8b802e81f88dafb40e/pyproject.toml#L13

I think we can assume it is not running on earlier pythons (and it probably won't for a lot of other reasons)

**Comment by a-detiste** [comment]
Good catch. I didn't noticed.

3.9 is plenty enough.

"mock" is fully fossilizied since 3.6

**Comment by kedder** [comment]
Thank you!

**Pull request merged**